src/explore-education-statistics-frontend/src/middleware/pages/redirectPages.ts

```diff
@@ -15,8 +15,8 @@ const cacheTime = getCacheTime();
 let cachedRedirects: CachedRedirects | undefined;
 
 const redirectPaths = {
-  methodologies: '/methodology',
-  publications: '/find-statistics',
+  methodologyRedirects: '/methodology',
+  publicationRedirects: '/find-statistics',
```

Comment:
I don't think we need to :)

I made sure to change the view-model on both the FE and BE in this PR so that merging this PR independently won't break anything

So my idea was to merge this PR first, and then once #5419 has been reviewed, merge that

Reply:
In that case, that's great organisation of them then! 🙂